src/chain_spec.rs

```diff
@@ -227,6 +227,7 @@ pub fn laminar_turbulence_latest_config() -> ChainSpec {
 
 const INITIAL_BALANCE: u128 = 1_000_000 * DOLLARS;
 const INITIAL_STAKING: u128 = 100_000 * DOLLARS;
+const HOURS: u64 = 60 * 60;
```

Comment:
We already have `HOURS` defined as block number, so maybe call this `HOURS_IN_SECONDS`

https://github.com/laminar-protocol/laminar-chain/blob/ccf5b0a1926ec3fa9d17c62fe2ff5c8e2b7fb985/runtime/src/constants.rs#L22